popsycle/synthetic.py

```diff
@@ -3514,32 +3665,400 @@ def generate_slurm_scripts(slurm_config_filename, popsycle_config_filename,
         print('Error: specified number of cores exceeds limit. Exiting...')
         return None
 
+    optional_cmds = ''
+
     # Pass along optional parameters if present
     if overwrite:
-        overwrite_cmd = '--overwrite'
-    else:
-        overwrite_cmd = ''
+        optional_cmds += '--overwrite '
 
-    if seed:
-        seed_cmd = '--seed=%i' % seed
-    else:
-        seed_cmd = ''
+    if seed is not None:
+        optional_cmds += '--seed=%i ' % seed
+
+    if skip_galaxia:
+        optional_cmds += '--skip-galaxia '
+
+    if skip_perform_pop_syn:
+        optional_cmds += '--skip-perform-pop-syn '
+
+    if skip_calc_events:
+        optional_cmds += '--skip-calc-events '
+
+    if skip_refine_events:
+        optional_cmds += '--skip-refine-events '
 
     # Populate the mpi_template specified inputs
     job_script = slurm_template.format(**locals())
 
     # Write the script to the path_run folder
-    script_filename = path_run + '/run_popsycle_%s_%s.sh' % (jobname,
-                                                             output_root)
+    script_filename = path_run + '/run_popsycle_%s.sh' % (jobname)
     with open(script_filename, 'w') as f:
         f.write(job_script)
 
     # Submit the job to disk
     if submitFlag:
+        os.chdir(path_run)
         stdout, stderr = execute('sbatch {0}'.format(script_filename))
-        print('** Standard Out **')
+        print('Submitted job {0} to {1} for {2} time'.format(script_filename,
+                                                             resource,
+                                                             walltime))
+        print('---- Standard Out')
         print(stdout)
-        print('** Standard Err **')
+        print('---- Standard Err')
         print(stderr)
+        print('')
```

Comment:
can you add either add some explanatory documentation to how this ubv to ztf conversion works and a visualization of the color color diagram, or eventually link to you paper if you will include it there?

Reply:
Yes, absolutely. I'm not including that in the paper, so I'll write up an explanation and put it in the docs folder.